assets/schema_input.json

```diff
@@ -16,7 +16,7 @@
             "contigs": {
                 "type": "string",
                 "format": "file-path",
-                "pattern": "^\\S+\\.f(ast|n)?a\\.gz$",
+                "pattern": "^\\S+\\.f(ast|n)?a(\\.gz)?$",
                 "errorMessage": "FASTA file containing assembled contigs, cannot contain spaces and must have extension '.fa.gz' or '.fasta.gz'"
```

Comment:
Can you update this message to include the other extensions (e.g., doesn't need to end in `.gz`)?

Reply:
Good catch! Updated the error message. d098aedad020658645d4b82272432b69a7583cf8